targets/python/templates/API.py.ejs

```diff
@@ -0,0 +1,44 @@
+import PlayFabHTTP
+import PlayFabSettings
+import PlayFabErrors
+import json
+
+def noneOrEmpty(str):
+    return str == "" or str == None
+
+<%- generateApiSummary("", api, "description")%><% for(var i in api.calls) { var apiCall = api.calls[i];
+%><%- generateApiSummary("", apiCall, "summary")
```

Comment:
Let's make sure that an extra line is added that has a url to the proper documentation for each/every API method.
Check out some other SDKs that generate that line.

Reply:
(Lua?)